### PR TITLE
moving workflow files to the right place

### PR DESCRIPTION
**Pull request by goern**
Depends-On: https://github.com/thoth-station/solver/pull/394

Signed-off-by: Christoph Görn <goern@redhat.com>

### CODE REVIEW

**Review by fridex — DISMISSED** [review]
:+1:

**Comment by CermakM** [comment]
@goern where did the `reconcile.sh` go?

File: reconcilers/solver/reconcile-solver-workflows [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# this might give an error, but we dont care...
+$OC_CMD delete workflow $($OC_CMD get workflows -o=jsonpath='{.items[?(@.status.phase=="Succeeded")].metadata.name}') || true
+
+while :
+do
+    echo "looping..."
+    for wf in $($OC_CMD get workflows -o=jsonpath='{.items[?(@.status.phase=="Error")].metadata.name}' ); do
+        argo retry $wf
+        sleep 7
+        $OC_CMD delete workflow $($OC_CMD get workflows -o=jsonpath='{.items[?(@.status.phase=="Succeeded")].metadata.name}')
```
Comment (CermakM) [inline reconcilers/solver/reconcile-solver-workflows]: This is something that we actually don't need to do ourselves. We can use the pod garbage collector for it and specify the strategy to be `OnWorkflowSuccess`

Comment (CermakM) [reply reconcilers/solver/reconcile-solver-workflows]: See https://github.com/argoproj/argo/blob/master/pkg/apis/workflow/v1alpha1/workflow_types.go#L58

Comment (goern) [reply reconcilers/solver/reconcile-solver-workflows]: uh, nice... I gonna change that

**Review by CermakM — DISMISSED** [review]

**Comment by goern** [comment]
regate

**Comment by goern** [comment]
regate

**Comment by goern** [comment]
recheck

File: reconcilers/solver/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM quay.io/thoth-station/thoth-ops:v0.15.0
```
Comment (harshad16) [inline reconcilers/solver/Dockerfile]:
```suggestion
FROM quay.io/thoth-station/thoth-ops:latest
```
As every time we would update thoth-ops, we will redirect latest to the updated version.

**Comment by harshad16** [comment]
LGTM :+1:  
PTAL @CermakM  and @fridex

**Review by CermakM — APPROVED** [review]

**Comment by goern** [comment]
regate

**Comment by goern** [comment]
regate

**Pull request merged**